src/endpoints/getAuditLogs.ts

```diff
@@ -93,15 +57,63 @@ export interface GetAuditLogQueryResultsOutput {
     nextToken?: string;
 }
 
-export const getAuditLogQueryResults = (params: GetAuditLogQueryResultsParams) => {
-    const { teamDeviceCredentials, ...payload } = params;
-    return requestTeamApi<GetAuditLogQueryResultsOutput>({
-        path: 'auditlogs-teamdevice/GetAuditLogQueryResults',
-        teamUuid: teamDeviceCredentials.uuid,
-        teamDeviceKeys: {
-            accessKey: teamDeviceCredentials.accessKey,
-            secretKey: teamDeviceCredentials.secretKey,
+export interface GetAuditLogQueryResultsRequest {
+    path: 'logs-teamdevice/GetAuditLogQueryResults';
+    input: GetAuditLogQueryResultsParams;
+    output: GetAuditLogQueryResultsOutput;
+}
+
+const MAX_RESULT = 1000;
+
+export const getAuditLogs = async (params: {
+    queryParams: StartAuditLogsQueryParams;
+    teamDeviceCredentials: TeamDeviceCredentials;
+}): Promise<GenericLog[]> => {
+    const { teamDeviceCredentials, queryParams } = params;
+
+    const api = await apiConnect({
+        isProduction: true,
+        enclavePcrList: [
```

Comment:
it would be nice to put the PCRs in a shared module/file because it's already used for the SSO endpoint too, so we have it in once place

Reply:
Moved the PCRs inside the module with a way to automatically switch between prod and staging and renamed the `isProduction` param for clarity